internal/docker/receiver/internal/metadata/generated_metrics.go

```diff
@@ -3946,7 +3946,7 @@ func (mb *MetricsBuilder) EmitForResource(rmo ...ResourceMetricsOption) {
 	rm := pmetric.NewResourceMetrics()
 	rm.SetSchemaUrl(conventions.SchemaURL)
 	ils := rm.ScopeMetrics().AppendEmpty()
-	ils.Scope().SetName("otelcol/dockerstatsreceiver")
+	ils.Scope().SetName("otelcol/docker/receiver")
```

Comment:
This is technically a breaking change, and should be at least mentioned in the changelog.